kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableClusteringSuite.scala

```diff
@@ -182,20 +182,113 @@ class DeltaTableClusteringSuite extends DeltaTableWriteSuiteBase {
     }
   }
 
-  test("update a table with clustering columns should be blocked") {
+  test("update a non-clustered table with clustering columns should succeed") {
```

Comment:
please test - update a partitioned table with clustering columns

This should fail right? I don't think we correctly do this check